pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: pwa
-version: 0.2.0
+version: 0.1.11
```

Comment:
Should probably also bump the SDK constraint to only support >=2.0. I noticed a change from `JSON` to `json`, which is breaking for dart 1.XX

Reply:
ack!